CONTRIBUTING.md

```diff
@@ -108,6 +108,164 @@ To learn about the viewer, run:
 cargo run -p rerun -- --help
 ```
 
+## Tests
+
+There are various kinds of automated tests throughout the repository.
+If not noted otherwise, all tests run automated on CI, however their frequency (per PR, on `main`, nightly, etc.) and platform coverage may vary.
+
+### Rust tests
+
+```sh
+cargo nextest run --all-targets --all-features
+cargo test --all-features --doc
+```
+or alternatively:
+```sh
+cargo test --all-targets --all-features
+```
```

Comment:
We should explain that `nextest` is something you have to install. I suggest switching them therefore:

```suggestion
	cargo test --all-targets --all-features

or alternatively (if you've [installed cargo nextest](https://nexte.st/)):

	cargo nextest run --all-targets --all-features
	cargo test --all-features --doc
```